core/imagedownloader.cpp

```diff
@@ -47,7 +41,6 @@ void ImageDownloader::load(QString filename, bool fromHash)
 
 	QNetworkRequest request(url);
 	request.setAttribute(QNetworkRequest::User, filename);
-	request.setAttribute(static_cast<QNetworkRequest::Attribute>(QNetworkRequest::User + 1), fromHash);
```

Comment:
This line, I am pretty sure I never wrote (nor have any idea what it does).

Reply:
> This line, I am pretty sure I never wrote (nor have any idea what it does).

That was me. :) It transports the `fromHash` flag to the request. Indeed, not pretty at all.